src/controls/shared/WidgetWithLabelUI.tsx

```diff
@@ -56,7 +56,7 @@ export const WidgetWithLabelUI = observer(function WidgetWithLabelUI_(p: {
         >
             <AnimatedSizeUI>
                 {/* LINE */}
-                <div tw={[isCollapsible && 'WIDGET-LINE', 'flex items-center gap-0.5']}>
+                <div tw={[WidgetBlockUI && 'WIDGET-LINE', 'flex items-center gap-0.5']}>
```

Comment:
This works, but it is not expected. Not sure why

Reply:
![image](https://github.com/rvion/CushyStudio/assets/703880/23a02712-8f08-4c94-b488-2d6c292bb9eb)

Reply:
Haha, this is what I wrote and was changed by bird_d to allow for widgets to override it; I think we actually need a bit of both; 

basically, widget now have 
1. a `line` part, 
2. 2. and a `block` part; 

line is shown inline, after the label, and block part is the block below.
if no block part, widget should never be collapsible, cause there is nothing to collapse.

as for the class names => those are temporary ugly stuff that should be renamed. `WIDGET-LINE` classname is wrongly named, and misleading; GOOD JOB managing to navigate around those undocumented things :)